build/test_update_ext_version.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import json
+import pathlib
+import pytest
+
+from update_ext_version import main as update_version, parse_version
```

Comment:
```suggestion
import update_ext_version
```

Rule of thumb is to import to the module, never objects in a module. Not only does it force you to use attribute access for such things and thus self-document where the object comes from, but it also avoids circular import issues.

Plus I just prefer it. 😁